### ISSUE TITLE
Extend metadata to include review information

### ISSUE THREAD

**Issue by cmungall** [OBOFoundry/OBOFoundry.github.io#1085]
We currently have reviews in noncomputable form here:

http://obofoundry.org/docs/CompletedReviews

We should instead put this in the metadata and derive the displays

E.g. foo.md:

```
...
reviews:
 - date: 2016
    url: <link to pdf... hopefully in future to a github ticket>
    status: pass
    comments: ...
```

 * date: REQUIRED can be EITHER 4-digit year OR ISO-8601 YYYY-MM-DD string
 * url: OPTIONAL <because we don't have old ones>
 * status: REQUIRED pass OR fail OR awaiting...?

Once this is in place we can dynamically

 * include the URL from the front table
 * include this in per-ontology pages
 * make the table in http://obofoundry.org/docs/CompletedReviews

Note that for a while the existing foundry tag in the md will be redundant but that is OK. We should have a consistency check.

**Comment by cmungall** [comment]
to be clear, the idea is to put the review metadata into the main metadata file.


ie in `ontology/bfo.md` we would have:

```
id: bfo
title: basic formal ontology
...
reviews:
 - date: 2016
    url: <link to pdf... hopefully in future to a github ticket>
    status: pass
    comments: ...
```

All: Do we want reviews to be a list? I think so, as we may have re-reviews. Should we also have some marker to say one of the reviews is the 'current' one? I think this is better than relying on ordering.

**Comment by nlharris** [comment]
what's the status of this?

**Comment by wdduncan** [comment]
@nlharris I fear this ticket has gone stale.

**Comment by nlharris** [comment]
Should we close it?

**Comment by wdduncan** [comment]
Let's ask @cmungall

**Comment by matentzn** [comment]
I dont think this should be closed. This is one of the things we need to take care of when we get new funding; maybe you can have a github tag that expresses this sentiment. No one will volunteer to do such a thing, but its a good item for a wishlist for when we are doing a serious overhaul of the site.

**Comment by matentzn** [comment]
But there is one practical aspect to this ticket we can do right now: add the reviews element to the json schema, and add the 8 reviews we currently have in the way chris suggests to the md files.

**Comment by nataled** [comment]
Closing since no longer relevant to current procedures.
